web/src/features/AppVersionHistory/AppVersionHistoryRow.tsx

```diff
@@ -180,12 +186,12 @@ function AppVersionHistoryRow(props: Props) {
   };
 
   const renderVersionAction = (version: Version) => {
-    const app = props.app;
+    const app = selectedApp;
```

Comment:
might not be necessary to reassign this to app

Reply:
I’ll clean that up. Just wanted to keep the changes limited for now.

Reply:
yep no worries!  😊